custom/enikshay/integrations/ninetyninedots/views.py

```diff
@@ -74,6 +76,16 @@ def update_patient_adherence(request, domain):
             message=("An error occurred updating the episode case after receiving a 99DOTS"
                      "adherence case for beneficiary {}. {}").format(beneficiary_id, e))
 
+    try:
+        for adherence_case in adherence_cases:
```

Comment:
I think the `factory.update_episode_adherence_properties()` will already take care of this (since that calls the reconciliation task). Otherwise, this should be fine.

Reply:
Oh i see, `update_single_episode` -> `EpisodeAdherenceUpdate.get_json()`

This is just getting the updates to be done on the episode case but does not run the `EpisodeAdherenceUpdate` updater/reconciliation. I am not sure if there is a way to merge the ledger update with `EpisodeAdherenceUpdate` and even if there is, it would be more efficient than doing ledger update as a follow up after the adherence update.